rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1125,19 +1125,21 @@ extern "C" rmw_ret_t rmw_init(const rmw_init_options_t * options, rmw_context_t
 
   context->instance_id = options->instance_id;
   context->implementation_identifier = eclipse_cyclonedds_identifier;
-  context->impl = nullptr;
 
-  if ((ret = rmw_init_options_copy(options, &context->options)) != RMW_RET_OK) {
-    return ret;
-  }
```

Comment:
I don't think that changing the order was needed

Reply:
It made easier, but it's not necessarily more correct. Changed.